dev/io.openliberty.org.jboss.resteasy.common/bnd.bnd

```diff
@@ -152,8 +152,8 @@ Private-Package: \
   org.apache.james.*
   
 instrument.ffdc: false
-instrument.disabled: false
-
+instrument.classesIncludes: **/*.class
+instrument.classesExcludes: org.jboss.resteasy.plugins.delegates.MediaTypeHeaderDelegate.class
```

Comment:
```suggestion
instrument.classesExcludes: org/jboss/resteasy/plugins/delegates/MediaTypeHeaderDelegate.class
```

I think these should be slashes rather than dots.